src/_locales/ru/messages.json

```diff
@@ -214,10 +214,10 @@
     "js_popup_whitelisted": { "message": "Сайт добавлен в белый список." },
     "js_popup_whitelisted_remove": { "message": "Удалить из белого списка" },
     "js_popup_audible": { "message": "Вкладка проигрывает аудио." },
-    "js_popup_form_input": { "message": "Вкладка получается данные с формы." },
+    "js_popup_form_input": { "message": "Вкладка имеет несохраненные поля ввода." },
     "js_popup_form_input_unpause": { "message": "Снять с паузы" },
     "js_popup_pinned": { "message": "Вкладка закреплена" },
-    "js_popup_temp_whitelist": { "message": "Пристановка вкладки поставлена на паузу." },
+    "js_popup_temp_whitelist": { "message": "Приостановка вкладки поставлена на паузу." },
     "js_popup_temp_whitelist_unpause": { "message": "Снять с паузы" },
```

Comment:
"Включить"

Reply:
And I double down "Включить" here, it could be "Возобновить" but "Включить" along with "Вкл./выкл. приостановку этой вкладки" and "Приостановка вкладки временно отключена." sounds and feels better, coz as I described above when u resuming ("Включить") suspension it's not temporarily ("временно") by itself but until user decides again to pause suspension.

Reply:
solved